patches.json

```diff
@@ -335,6 +335,11 @@
       "3.4.1": "BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch"
     }
   },
+  "paypal/module-braintree-core": {
+    "Fix Braintree Settlement report error": {
+      "4.1.0": "BUNDLE-2683__braintree_settlement_report_fix__4.1.0.patch"
```

Comment:
The patch file path is different to real file:
`BUNDLE-2683__braintree_settlement_report_fix__4.1.0.patch`
vs
`BUNDLE-2683__braintree_settlement_report_fix__4.0.1.patch`
Please change

Reply:
Thanks. 4.1.0 is correct.